Fix: Allow the use of MdsException(string) in python

tfredian opened the pull request:
The definition of the MdsException class in python was changed
to be equivalent to the MDSplusException class which is used to
create exceptions from numeric status codes which are returned by
MDSplus C functions. This prevents the ability for applications
to create MdsExceptions using a string argument. This change
makes MdsException a subclass of Exception and the MDSplusException
class be a subclass of MdsException.

This should fix: https://github.com/MDSplus/mdsplus/issues/1388

tfredian committed:
Fix: Allow the use of MdsException(string) in python

The definition of the MdsException class in python was changed
to be equivalent to the MDSplusException class which is used to
create exceptions from numeric status codes which are returned by
MDSplus C functions. This prevents the ability for applications
to create MdsExceptions using a string argument. This change
makes MdsException a subclass of Exception and the MDSplusException
class be a subclass of MdsException.

eldond commented:
I think this would close #1388.

tfredian commented:
retest this please

tfredian commented:
retest this please

tfredian commented:
retest this please

zack-vii reviewed (dismissed):
should not break anything

tfredian committed:
Build: Use jars subdirectory in source directory tree

Instead specifying a directory the jars are placed in src/jars
so only --jars options is needed.

tfredian commented:
retest this please

tfredian commented:
retest this please
